axon-mongo-spring-boot-starter/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2010-2022. Axon Framework
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>axon-mongo-parent</artifactId>
```

Comment:
Unsure whether we should, in all honesty, but the other starter dependencies we provide have a parent pointing to `groupId` `org.springframework.boot` with `artifactId` `spring-boot-starters`.
As far as I can tell, that would help with depending on springs other starters...
Perhaps @abuijze can tell why the original starter had `spring-boot-starters` as the parent?

Reply:
Interesting, I looked up a few other starters, but none of them have `spring-boot-starters` as parent.

Reply:
Just chatted with Allard on this. We assume it's an old convention, as back in the day, using `spring-boot-starters` was expected.
Googled about a bit, but I did come to the same conclusion as you: I cannot find the usage of `spring-boot-starters` as the parent dependency for a starter.
So, let's disregard this then.